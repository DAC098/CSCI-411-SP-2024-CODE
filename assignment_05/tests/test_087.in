865 779
15
24
32
37
48
56
72
94
104
114
120
146
158
176
192
197
216
274
293
301
321
334
338
361
371
397
406
411
430
442
446
451
454
471
476
481
506
514
523
580
584
596
615
615
626
628
631
637
720
725
729
732
786
796
798
824
846
848
853
866
866
869
873
889
893
896
901
920
922
943
959
984
1005
1009
1017
1030
1030
1048
1049
1058
1070
1102
1115
1117
1128
1140
1166
1167
1170
1173
1176
1190
1214
1234
1239
1249
1259
1264
1272
1286
1292
1294
1296
1303
1304
1307
1315
1337
1340
1346
1347
1397
1440
1445
1446
1460
1465
1467
1478
1486
1500
1502
1520
1528
1530
1534
1535
1540
1551
1552
1568
1574
1584
1659
1664
1693
1697
1698
1719
1719
1760
1790
1801
1809
1823
1834
1845
1857
1870
1873
1879
1894
1898
1899
1903
1908
1913
1915
1922
1940
1946
1946
1953
1967
1977
1998
2001
2005
2019
2022
2028
2034
2037
2043
2065
2074
2076
2086
2089
2093
2097
2097
2110
2140
2169
2180
2181
2184
2187
2196
2197
2219
2219
2229
2242
2251
2252
2266
2278
2280
2299
2306
2349
2350
2351
2358
2361
2365
2380
2391
2393
2402
2409
2419
2430
2437
2445
2449
2451
2452
2469
2479
2487
2518
2524
2527
2536
2547
2563
2590
2647
2659
2659
2661
2668
2677
2679
2680
2685
2713
2724
2750
2780
2782
2798
2816
2848
2853
2854
2872
2881
2891
2906
2919
2921
2967
2971
2975
2984
2988
2992
2997
3004
3025
3060
3062
3081
3085
3089
3094
3098
3132
3132
3136
3145
3148
3162
3164
3167
3169
3173
3181
3217
3234
3246
3256
3256
3257
3264
3275
3276
3283
3289
3310
3327
3333
3344
3345
3345
3345
3349
3361
3366
3372
3385
3395
3401
3421
3438
3441
3445
3455
3461
3479
3504
3505
3533
3550
3554
3561
3570
3572
3577
3609
3626
3637
3652
3657
3678
3706
3715
3744
3766
3769
3772
3782
3823
3824
3824
3842
3843
3850
3855
3877
3884
3887
3897
3900
3906
3931
3938
3941
3967
3986
3996
4023
4032
4053
4053
4054
4076
4077
4088
4094
4098
4108
4113
4115
4117
4117
4124
4136
4138
4139
4155
4156
4185
4187
4225
4260
4262
4269
4272
4323
4328
4333
4337
4346
4360
4368
4405
4413
4422
4439
4445
4450
4457
4465
4485
4509
4524
4538
4538
4565
4583
4590
4590
4593
4613
4615
4635
4650
4651
4658
4681
4682
4706
4735
4752
4755
4758
4769
4780
4827
4837
4843
4854
4856
4867
4872
4908
4909
4923
4926
4930
4957
4962
4974
4984
4990
4991
4996
5008
5019
5028
5028
5047
5051
5052
5064
5072
5077
5080
5082
5090
5099
5137
5139
5139
5164
5166
5167
5178
5180
5184
5188
5218
5323
5352
5354
5365
5365
5372
5382
5400
5410
5414
5417
5421
5426
5428
5447
5447
5469
5505
5506
5532
5534
5536
5537
5551
5553
5589
5605
5618
5623
5631
5634
5637
5654
5657
5660
5673
5680
5682
5695
5698
5717
5781
5781
5796
5799
5817
5829
5830
5834
5849
5851
5878
5882
5887
5904
5949
5991
5997
6003
6018
6021
6032
6043
6055
6062
6065
6074
6084
6085
6124
6157
6157
6157
6164
6174
6177
6184
6204
6206
6207
6211
6221
6230
6254
6287
6290
6303
6304
6310
6329
6380
6394
6424
6457
6470
6477
6479
6487
6490
6503
6522
6524
6581
6609
6643
6643
6655
6692
6699
6727
6727
6738
6762
6765
6771
6772
6778
6820
6836
6836
6857
6886
6898
6902
6905
6907
6942
6952
7009
7014
7018
7049
7060
7062
7062
7066
7068
7072
7086
7096
7108
7120
7126
7139
7160
7162
7163
7174
7178
7182
7190
7198
7199
7218
7234
7250
7251
7283
7317
7319
7330
7342
7356
7356
7365
7370
7399
7401
7412
7426
7440
7464
7466
7482
7501
7515
7521
7530
7539
7549
7556
7575
7580
7583
7588
7602
7603
7646
7655
7660
7671
7685
7699
7749
7749
7761
7781
7796
7804
7804
7847
7849
7865
7885
7896
7897
7904
7908
7917
7921
7931
7943
7952
7961
7977
7993
7994
8035
8036
8049
8054
8066
8076
8111
8135
8153
8154
8174
8178
8178
8196
8202
8219
8245
8265
8269
8272
8293
8303
8308
8313
8320
8322
8330
8336
8340
8342
8360
8362
8363
8378
8381
8438
8440
8449
8464
8474
8479
8480
8485
8510
8511
8551
8575
8602
8607
8623
8636
8641
8659
8662
8687
8689
8691
8696
8697
8703
8710
8712
8715
8749
8750
8766
8781
8784
8794
8794
8806
8821
8821
8828
8833
8833
8833
8839
8872
8884
8894
8897
8909
8910
8915
8916
8947
8948
8994
9000
9011
9017
9019
9032
9032
9036
9041
9050
9056
9065
9084
9100
9128
9137
9168
9169
9170
9192
9205
9215
9237
9253
9255
9256
9272
9285
9286
9287
9287
9294
9296
9306
9312
9329
9337
9341
9345
9360
9373
9375
9397
9420
9432
9435
9480
9490
9490
9495
9513
9526
9553
9555
9581
9586
9587
9599
9644
9664
9717
9718
9734
9741
9743
9744
9756
9761
9771
9784
9814
9827
9837
9840
9854
9883
9885
9888
9907
9908
9912
9918
9920
9924
9932
9946
9948
9957
9957
9961
9963
9995
9999
