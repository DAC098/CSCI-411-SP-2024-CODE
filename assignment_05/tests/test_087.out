Enter the number of hotels and the ideal number of miles to travel per day: Enter 865 hotel distances each on a separate line: Hotels to visit: 53 129 202 267 343 411 475 542 594 660 723 795 865 
Total penalty: 2210
